packages/server/src/utilities/rowProcessor/index.ts

```diff
@@ -88,7 +97,36 @@ export async function processAutoColumn(
         break
     }
   }
-  return { table, row }
+}
+
+async function processDeafultValues(table: Table, row: Row) {
+  const ctx: { ["Current User"]?: User; user?: User } = {}
+
+  const identity = context.getIdentity()
+  if (identity) {
+    if (identity._id && identity.type === IdentityType.USER) {
+      const user = await cache.user.getUser(identity._id)
+      delete user.password
+
+      ctx["Current User"] = user
+      ctx.user = user
+    }
+  }
+
+  for (let [key, schema] of Object.entries(table.schema)) {
+    if ("default" in schema && schema.default != null && row[key] == null) {
```

Comment:
I'm not sure I understand what you mean.

Reply:
We only check if it's null, but not undefined

Reply:
Ah, this confused me at first as well but it's a pattern I see used a lot in the codebase. Because it's `!=` and not `!==` it's a coerced comparison.

```javascript
> let foo = null
undefined
> foo
null
> foo == null
true
> foo == undefined
true
> foo != null
false
> foo != undefined
false
```